docs/fsharp/tour.md

```diff
@@ -49,11 +49,11 @@ It's important to note that because `struct` tuples are value types, they cannot
 
 ## Pipelines and Composition
 
-Pipe operators (`|>`, `<|`, `||>`, `<||`, `|||>`, `<|||`) and composition operators (`>>` and `<<`) are used extensively when processing data in F#.  These operators are functions which allow you to establish "pipelines" of functions in a flexible manner.  The following example walks through how you could take advantage of these operators to build a simple functional pipeline.
+Pipe operators such as `|>` are used extensively when processing data in F#. These operators are functions which allow you to establish "pipelines" of functions in a flexible manner.  The following example walks through how you could take advantage of these operators to build a simple functional pipeline.
```

Comment:
are functions which allow  -> are functions that allow 

remove extra space between sentences

could take -> can take

end the paragraph with :